ConsensusMaker.py

```diff
@@ -424,10 +426,13 @@ def main():
             UP += 1
         else:
             b = consensusDict.pop(consTag)
+            b.qname = removeReadNumberFromQName(b.qname)
+            b.qname = "%d:%s" % (outputReadNum, a.qname)
             # keep flags: read unmapped, read reverse strand, not primary alignment, vendor QC, PCR duplicate, supplementary
             b.flag = b.flag & (4 | 16 | 256 | 512 | 1024 | 2048)
-            b.qname = removeReadNumberFromQName(b.qname)
-            b.qname = "%d:%s" % (outputReadNum, b.qname)
+            b.mrnm = -1
```

Comment:
seriously pysam, without this makes the default aligned read a paired end read even if the flags are set correctly